pkg/destroy/openstack/openstack.go

```diff
@@ -671,18 +671,25 @@ func deleteLoadBalancers(opts *clientconfig.ClientOpts, filter Filter, logger lo
 
 	conn, err := clientconfig.NewServiceClient("load-balancer", opts)
 	if err != nil {
-		logger.Debugf("%v", err)
-		return true, nil
+		// Ignore the error if Octavia is not available for the cloud
+		if _, ok := err.(*gophercloud.ErrEndpointNotFound); ok {
+			logger.Debug("Skip load balancer deletion because Octavia endpoint is not found")
+			return true, nil
+		}
+		logger.Errorf("%v", err)
```

Comment:
why not just `logger.Error` ?

Reply:
This pattern is used everywhere in the file and I decided to keep it for consistency